fiftyone/core/view.py

```
@@ -389,6 +389,22 @@ def default_skeleton(self):
     def default_skeleton(self, skeleton):
         self._root_dataset.default_skeleton = skeleton
 
+    @property
+    def camera_intrinsics(self):
+        return self._root_dataset.camera_intrinsics
+
+    @camera_intrinsics.setter
+    def camera_intrinsics(self, intrinsics):
```

Ruff (0.14.10)

Missing return type annotation for private function `camera_intrinsics`

Add return type annotation: `None`

(ANN202)